portality/store.py

```diff
@@ -217,12 +222,17 @@ def delete_file(self, container_id, target_name):
             else:
                 shutil.rmtree(cpath)
 
+    def size(self, container_id, target_name):
```

Comment:
Is this method used anywhere else than the tests?

Reply:
no, just needed for a test, but could come in handy in the future.  StoreLocal specific.